code/game/objects/items/weapons/grenades/clusterbuster.dm

```diff
@@ -126,3 +126,18 @@
 /obj/item/weapon/grenade/clusterbuster/clf3
 	name = "WELCOME TO HELL"
 	payload = /obj/item/weapon/grenade/chem_grenade/clf3
+
+/obj/item/weapon/grenade/clusterbuster/plasma
+ 	name = "Plasma Cluster Grenade"
+ 	desc = "For when everything needs to die in a fire."
+ 	payload = /obj/item/weapon/grenade/gas
+
+/obj/item/weapon/grenade/clusterbuster/n2o
+ 	name = "N2O Cluster Grenade"
+ 	desc = "For when you need to knock out EVERYONE."
+ 	payload = /obj/item/weapon/grenade/gas/knockout
+
+/obj/item/weapon/grenade/clusterbuster/freon
+ 	name = "Freon Cluster Grenade"
+ 	desc = "For when you need everyone to chill out."
```

Comment:
The dinosaurs were killed by a big ball of fire and death that caused cold, not a big ball of cold that caused death.